### PR TITLE
[guilib] Add container.listitemabsolute.infolabel

### PR DESCRIPTION
**Pull request by BigNoid**
This adds `Container.ListItemAbsolute.[INFOLABEL]` where `ListItemAbsolute` is the item position irrelevant of cursor position unlike `ListItemPosition`
I need this to be able to scroll two lists at the same time correctly in leanback.
@mkortstiege @phil65

### CODE REVIEW

**Comment by phil65** [comment]
Sounds useful, I dont really like the naming though. Perhaps ListItemAbsolute?

**Comment by BigNoid** [comment]
That has crossed my mind too, but ```listitemposition``` is refered to as absolute in code/wiki. This would be more like the item id in static content, hence the name. No real strong opinion on that though.

**Comment by phil65** [comment]
Yeah the wiki entry for ListItemPosition is wrong though, in the beginning I thaught that listitemposition would behave like this one. I would go with ListItemAbsolute.

**Comment by BigNoid** [comment]
Changed to ```ListItem.Absolute```

**Comment by phil65** [comment]
Cool, thx.

**Comment by BigNoid** [comment]
jenkins build this please

**Comment by mkortstiege** [comment]
Looks good to me but i think we should merge `listitem`, `listitemposition`, `listitemnowrap` and `listitemabsolute` to reduce dupe code where possible.

**Comment by phil65** [comment]
agreed, they all look very similar.

**Comment by BigNoid** [comment]
Added a separate commit that reduces dupe code.

File: xbmc/GUIInfoManager.cpp [hunk 1]
```diff
@@ -1203,34 +1203,23 @@ int CGUIInfoManager::TranslateSingleString(const std::string &strCondition, bool
         return AddMultiInfo(GUIInfo(CONTAINER_SORT_DIRECTION, order));
       }
     }
-    else if (cat.name == "listitem")
+    else if (cat.name == "listitem" || cat.name == "listitemposition" || cat.name == "listitemnowrap" || cat.name == "listitemabsolute")
```
Comment (mkortstiege) [inline xbmc/GUIInfoManager.cpp]: Line is a bit long ;) Please wrap.

File: xbmc/guiinfo/GUIInfoLabels.h [hunk 2]
```diff
@@ -736,4 +736,5 @@
 // Stored in the top 8 bits of GUIInfo::m_data1
 // therefore we only have room for 8 flags
 #define INFOFLAG_LISTITEM_WRAP        ((uint32_t) (1 << 25))  // Wrap ListItem lookups
-#define INFOFLAG_LISTITEM_POSITION    ((uint32_t) (1 << 26))  // Absolute ListItem lookups
+#define INFOFLAG_LISTITEM_POSITION    ((uint32_t) (1 << 26))  // ListItem lookups based on cursor position
+#define INFOFLAG_LISTITEM_ABSOLUTE    ((uint32_t) (1 << 26))  // Absolute ListItem lookups
```
Comment (mkortstiege) [inline xbmc/guiinfo/GUIInfoLabels.h]: c/p error? POSITION and ABSOLUTE using the same.

**Comment by BigNoid** [comment]
@mkortstiege updated

**Comment by mkortstiege** [comment]
jenkins build this please.

**Pull request merged**